frontend/assets/stylesheets/components/_faqs.scss

```diff
@@ -4,9 +4,43 @@
 
 .faq {
     padding-bottom: $gs-gutter;
+    input[type=checkbox] {
+        display: none;
+    }
+    input[type=checkbox] ~ .faq__answer {
+        max-height: 0;
+        opacity: 0;
+        overflow: hidden;
+        transition: all 0.8s ease;
```

Comment:
The transition is nice, but it feels slightly slow at the moment to me 🏎. What do you think about dropping this to `0.1s`?